lib/entities/artifact/ArtSlotInfo.h

```diff
@@ -21,6 +21,7 @@ struct DLL_LINKAGE ArtSlotInfo : public GameCallbackHolder
 	ArtifactInstanceID artifactID;
 	bool locked = false; //if locked, then artifact points to the combined artifact
 
+	ArtSlotInfo() = delete;
```

Comment:
Not against it, but why is it needed? Default constructor should be only created if there are no other constructrors